MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -182,8 +189,10 @@ await using (cacheStream.ConfigureAwait(false))
                     }
                 }
 
-                var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
-                using var response = await httpClient.GetAsync(url, cancellationToken).ConfigureAwait(false);
+                using var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
```

Comment:
I'll note that the compiler was complaining about an ignored `IDisposable` if i in-lined the HTTP client in the ternary, and without the `using` when it was used in the ternary, so I added it as a separate line above. Feel free to tell me a better way to do it if any.

Reply:
HttpClients shouldn't be disposed regardless of any warnings you may see. It is effectively a no-op when the client is created using the factory